app/js/bootstrap.js

```diff
@@ -2,11 +2,19 @@
 
 require.config({
   // ReactDOM expects "react" module to be defined, but it is not.
-  map: {
-    '*': {
-      'react': 'components/react',
-    },
-  },
+  map: { '*': { 'react': 'components/react' } },
 });
 
-require(['js/app.js']);
+const polyfills = [];
```

Comment:
Idea is to load polyfills only when we need them (and Edge will definitely require more polyfills), what do you think?